src/core/store/interface.ts

```diff
@@ -38,50 +35,31 @@ export interface PassThroughCallback {
  * an put function, a dequeue function and a get function.
  */
 export interface StoreInterface {
+  /**
+   * put() takes a batch file to store with the chosen hosting solution and
+   * returns the URL of the file once stored.
+   *
+   * @param   targetPath  The path to and name of file
+   * @param   content The file object to store on the chosen hosting solution
+   * @returns       The URI of the hosted file
+   */
   put?: (targetPath: string, content: Content) => Promise<URL>;
+
+  /**
+   * get() fetches a batch file from the given target path.
+   *
+   * @param   targetPath  The path to and name of file.
+   * @returns   The batch file fetched from the given URI
+   */
   get?: (targetPath: string) => Promise<File>;
```

Comment:
discuss removing. Discuss having different interface which contains union of other interfaces.

Reply:
I'm up for removing it

Reply:
Removed.